charter.html

```diff
@@ -237,11 +255,11 @@ <h3 id="out-of-scope">
             algorithms.
```

Comment:
We excluded training capability from the original charter. Do we want to rediscuss that aspect of the spec for webnn v2?

Reply:
Thanks for raising this. I have opened #27 to solicit input on training capability. Please provide your feedback in that issue to feed into a broader WG discussion. We still have time to in-scope this if there is adequate interest.